src/Microsoft.TestPlatform.CommunicationUtilities/TestRequestSender.cs

```diff
@@ -52,6 +52,7 @@ public class TestRequestSender : ITestRequestSender
         // that implies host is using version 1.
         private int protocolVersion = 1;
 
+        // Also check TestRequestHandler.
```

Comment:
Nit: We probably need to clarify this comment. It took a while for me to understand it.

```suggestion
        // Must be in sync with the highest supported version in src/Microsoft.TestPlatform.CrossPlatEngine/EventHandlers/TestRequestHandler.cs file.
```

Reply:
Done.